### ISSUE TITLE
[a11y] Keyboard navigation does not work as expected in App Exclusions screen

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#6467]
**Mozilla VPN affected versions:**

* 2.15.0 (2.202303231059);

**Platform tested:**

* all;

**Prerequisites:**

* Have the Mozilla VPN opened;
* Have an account with subscription for Mozilla VPN;
* Be signed in;

**Steps to reproduce:**

1. Go to Settings - App exclusions;
1. Navigate to one app using the keyboard;
1. Check one app using the keyboard;
1. Continue the navigation using the keyboard;

**Expected results:**

* User is able to select another app and to add it to the exclusion list;

**Actual results:**

* After selecting one app, the user is redirected to the “Back” option.

**Notes:**

* User is redirected to the “Back” option after each exclusion check.
* Not a regression of App exclusion revamp;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-4447)
┆Reporter: Valentina Virlics

**Comment by data-sync-user** [comment]
➤ Lesley Norton commented:

Thanks for filing Valentina Virlics . Is the move back to the ‘back’ button after making a selection an expected a11y ux treatment? And if so should we be doing this in other lists? If not, I’m not sure that I agree that the user should be moved from the selected checkbox to the ‘back’ button because what if they want to check/uncheck multiple items?

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Lesley Norton 

Reproducing for Languages as well, after selecting one language user is moved ot Back option, while keyboard navigation.

Not reproducing for Privacy features, Notifications, etc,

I would expect user to be able to select/unselect multiple options before being redirected to leave that screen.

**Comment by data-sync-user** [comment]
➤ Lesley Norton commented:

Thanks Valentina Virlics I misunderstood the ticket.
